cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -96,37 +107,43 @@ func (r *KubeArchiveConfigReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
-	_, err := r.reconcileA14eServiceAccount(ctx, kaconfig)
+	cm, err := r.reconcileFilterConfigMap(ctx, kaconfig)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	resources := r.parseConfigMap(ctx, cm)
+
+	_, err = r.reconcileA14eServiceAccount(ctx, kaconfig)
 	if err != nil {
 		return ctrl.Result{}, err
 	}
 
-	_, err = r.reconcileA14eRole(ctx, kaconfig)
+	clusterrole, err := r.reconcileA14eRole(ctx, kaconfig, resources)
 	if err != nil {
 		return ctrl.Result{}, err
 	}
 
-	_, err = r.reconcileA14eRoleBinding(ctx, kaconfig)
+	_, err = r.reconcileA14eRoleBinding(ctx, kaconfig, clusterrole)
```

Comment:
Just wondering and a bit offtopic for the MR
Considering A14e for "ApiServerSource", but "piServerSourc" are 13 letters, so, shouldn't be A13e?

Reply:
Good catch. I've changed to A13e.  I must have mis-counted.